custom/enikshay/forms.py

```diff
@@ -53,6 +53,7 @@ class DataDumpTaskForm(forms.Form):
                             for choice in permitted_tasks]
     ))
     case_type = forms.CharField(label="Case Type")
+    dump_title = forms.CharField(label="Dump Title")
```

Comment:
this is not needed now. I added this [here](https://github.com/dimagi/commcare-hq/pull/19555/files#diff-1b97b1cb3c6b38074e42ad58b8eef419R55) using the task name.